File: C++17/백준/Bronze/24075. 計算 （Calculation）/計算 （Calculation）.cc
#include <iostream>
using namespace std;

int main () {
    int a, b;
    cin >> a >> b;
    
    int sum_val = a + b;
    int sub_val  = a - b;
    
    if (sum_val > sub_val) {
        cout << sum_val << endl;
        cout << sub_val << endl;
    } else {
        cout << sub_val << endl;
        cout << sum_val << endl;
    }
    
    return 0;
}

File: Java/백준/Bronze/24075. 計算 （Calculation）/計算 （Calculation）.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int a = sc.nextInt();
        int b = sc.nextInt();
        
        int sum_val = a + b;
        int sub_val = a - b;
        
        System.out.println(Math.max(sum_val, sub_val));
        System.out.println(Math.min(sum_val, sub_val));
        
        sc.close();
    }
}